### ISSUE TITLE
Explore "approximate" gradients

### ISSUE THREAD

**Issue by WardLT** [exalearn/fast-finetuned-forcefields#4]
We cannot get gradients with MP2 in NWChemEx (yet!) and they are additional expense anyway.

What happens if we train without gradients or using numerical approximations of 1 or more directions.

**Comment by WardLT** [comment]
Update from SPEC (excited states): 

- CC has 95% complete gradients (missing just one term). Are they "close enough" to be useful? [Q for PNNL crew: is CC good for water]?
    - They have it for "singles and doubles," which is generally accepted to be better than MP2
    - Not yet sure what the timeline for it is (check in with them in mid/late-Jan about it)
